src/main.js

```diff
@@ -0,0 +1,10 @@
+import App from './App.svelte';
```

Comment:
Hi @ClaraChaouat,

It looks like some files are missing from git, did you commit and push everything?

Some files missing: `package.json`.

In these type of projects that require some setup, it's a common practice to leave instructions in the `README.md`. You can look for examples out there of nice readmes. For yours a description and instructions is enough for starting :)

Here are some [awesome readme's](https://github.com/matiassingers/awesome-readme) from the community

Reply:
Hi Sandrina, 

I was wondering what files you needed to have a look at the project. So, I got that you need the package.json file. Do you need another one? All of them maybe? I didn't want to push all of them because I wasn't sure I was relevant.

Reply:
In a project usually, you commit all the files, except `node_modules` and cache folders `ex __svelte__` :)